src/layman/geoserver_proxy.py

```diff
@@ -28,7 +28,10 @@ def ensure_wfs_t_attributes(binary_data):
     try:
         xml_tree = ET.XML(binary_data)
         version = xml_tree.get('version')[0:4]
-        if version not in ["2.0.", "1.0.", "1.1."] or xml_tree.get('service').upper() != "WFS":
+        service = xml_tree.get('service').upper()
+        if service != 'WFS':
+            return
+        if version not in ["2.0.", "1.0.", "1.1."]:
             app.logger.warning(f"WFS Proxy: only xml version 2.0, 1.1, 1.0 and WFS service are supported. Request "
```

Comment:
Remove `and WFS service are supported` and `}, service={xml_tree.get('service')}`